File: addBorder/addBorder.js
function addBorder(a) {
    let r = a.length,
        c = a[0].length,
        str = "*".repeat(c + 2);
    for (let i = 0; i < r; i++) {
        a[i] = "*".concat(a[i], "*")
    };
    a.unshift(str);
    a.push(str);
    return a
}
console.log(addBorder(["a"]));
console.log(addBorder(["abc", "ded"]));

File: Completed/js/addBorder/addBorder.py
def addBorder(a):
    r = len(a)
    c = len(a[0])
    wall = "".join("*"*(c+2))
    for i in range(r):
       a[i] = "*" + a[i] + "*"
    a.insert(0,wall)
    a.append(wall)
    return a

print(addBorder(["a"]));
print(addBorder(["abc", "ded"]));